vignettes/tlf-primary.Rmd

```diff
@@ -78,13 +78,18 @@ t <- tplyr_table(adas, TRTP) %>%
     group_desc(CHG,  where= AVISITN == 24, by = "Change from Baseline")
   )
   
+hdr <- unique(adas$TRTP)
+hdr_ext <- sapply(hdr, FUN = function(x) paste0("|", x, "\\line(N=**", x, "**)"), USE.NAMES = FALSE)
```

Comment:
@SHAESEN2 this is already defensively coded in Tplyr:

```r
 paste0("|Placebo\\line(N=**Placebo**)| Xanomeline High Dose\\line(N=**Xanomeline High Dose**) ", 
           "| Xanomeline Low\\line Dose\\line(N=**Xanomeline Low Dose**)")
```

The `**Xanomeline High Dose**, **Placebo**` etc. text automatically captures the header N's as calculated when the table is built. 

The output in this case was accurate, unless I'm missing something. Was there another specific comment you're trying to work around, besides the fact that I just had the order of the headers mixed up in my original output?

Reply:
Hey @mstackhouse I think my fork was behind this repo, because I did not see this update. The defensive part is that we should not hard code headers, but extract them from the data so that these mistakes don't occur.

Reply:
Ok I see what you're doing now. Sorry - at first I interpreted as you calculating the header N's yourself because I didn't look at the code closely enough.

Thanks - this would be a good update to provide tooling in Tplyr as well.